ide/che-core-ide-api/src/main/java/org/eclipse/che/ide/api/machine/DevMachineServerProperties.java

```diff
@@ -0,0 +1,47 @@
+/*******************************************************************************
+ * Copyright (c) 2012-2016 Codenvy, S.A., Red Hat, Inc.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *   Mario Loriedo - initial API and implementation
+ *******************************************************************************/
+package org.eclipse.che.ide.api.machine;
+
+import org.eclipse.che.api.core.model.machine.ServerProperties;
+
+/**
+ * Describe development machine server instance.
+ * {@link ServerProperties}
+ *
+ * @author Mario Loriedo
+ */
+public class DevMachineServerProperties implements ServerProperties {
+
+    private final String path;
+    private final String internalAddress;
+    private final String internalUrl;
+
+    public DevMachineServerProperties(ServerProperties dto) {
```

Comment:
Argument doesn't have to be DTO object. Please rename parameter to not confuse developers.